kodilla-patterns2/src/main/java/com/kodilla/patterns2/facade/api/OrderFacadeWatcher.java

```diff
@@ -0,0 +1,19 @@
+package com.kodilla.patterns2.facade.api;
+
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class OrderFacadeWatcher {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrderFacadeWatcher.class);
+
+    @Before("execution(* com.kodilla.patterns2.facade.api.OrderFacade.processOrder(..))")
+    public void logEvent() {
+        LOGGER.info("Method processOrder() started");
```

Comment:
Fajnie by było, abyś dodał jakieś dodatkowe informacje o tym logowaniu, chociażby co to za order przetwarzać. Taki log nic nam nie powie w przyszłości.